zrml/neo-swaps/src/tests/sell.rs

```diff
@@ -243,23 +243,110 @@ fn sell_fails_on_asset_not_found(market_type: MarketType) {
 }
 
 #[test]
-fn sell_fails_on_numerical_limits() {
+fn sell_fails_if_amount_in_is_greater_than_numerical_threshold() {
```

Comment:
Can you show the reader the numerical threshold as a number here and assert it against the amount_in?

Reply:
012e595e0efa32b4be0865bf4a4a9831bc36c761